epub34/authoring/index.html

```diff
@@ -12022,7 +12023,12 @@ <h2>Change log</h2>
 			<details id="changes-epub-33" open="open">
 				<summary>Substantive changes since <a href="https://www.w3.org/TR/epub-33/">EPUB 3.3</a></summary>
 				<ul>
-					<li>08-Apr-2025: Added early support for HTML syntax.</li>
+					<li>26-May-2025: Added support for the <code>application/x-font-ttf</code> media type. 
```

Comment:
I know this is getting nitpicky, but "support" sounds like we're adding TTF font support. I'd maybe deemphasize the scope of the change a bit:

```suggestion
					<li>26-May-2025: Added <code>application/x-font-ttf</code> to the list of core media types for identifying TTF fonts. 
```